build-zip.sh

```diff
@@ -38,62 +31,65 @@ function get_repo_base_url() {
   esac
 
   echo $BASE_URL
-} && export -f get_repo_base_url
+}
```

Comment:
Just... why?

Reply:
https://github.com/zaoqi/flashable-zip_microG/blob/59d66c4c6fb814dd5106162cd69a06e351ddfc04/build-zip.sh#L112

Reply:
I meant why rewritting

Reply:
to support complex configurations and remove GNU awk, xargs requirements